cluster/libexec/wwinit/50-tftp.init

```diff
@@ -25,4 +25,11 @@ if wwpackage_check tftp-server tftpd tftpd-hpa; then
         exit 0
     fi
 fi
+
+if wwpackage_check dnsmasq; then
```

Comment:
We should only check for the dnsmasq package if its selected in provision.conf. Also vice versa, ISC should only be checked for it its selected. See https://github.com/warewulf/warewulf3/blob/development/common/libexec/wwinit/10-database.init#L143 for an example of how we deal with this for database types.